.github/workflows/release.yml

```diff
@@ -69,7 +69,7 @@ jobs:
           path: "dist/"
 
       - name: "Publish package distributions to PyPI"
-        uses: "pypa/gh-action-pypi-publish@ed0c53931b1dc9bd32cbe73a98c7f6766f8a527e"  # v1.13.0
+        uses: "pypa/gh-action-pypi-publish@ed0c53931b1dc9bd32cbe73a98c7f6766f8a527e" # v1.13.0
```

Comment:
Diffsync must be different, but we enforce two spaces between content and code comment with yamllint.